bump.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+# ----------- Basic Checking -----------
+
+# Check that 1 arg has been supplied
+if [ "$#" != "1" ]; then
+	echo "[-] Exactly one argument is needed."
+	echo '[?] Example: `./bump.sh 0.2.6`'
+	exit 1
+fi
+
+# Check that required binaries are installed
+for package in semver git npm; do
+	if [ ! -f "/usr/bin/$package" ]; then
+		echo "[-] `$package` is needed to bump version"
+		exit 1
+	fi
+done
+
+# Check that supplied version is semantically correct
+if [ ! "$1" = "$(/usr/bin/semver $1)" ]; then
+	echo "[-] $1 is not a valid version number according to semver"
+	exit 1
+fi
+
+# Check that git working directory is clean...
+if [ -n "$(git status -s --untracked-files=no)" ]; then
+	echo "[-] git working directory isn't clean"
+	exit 1
+fi
+
+# ----------- Variable definition -----------
+
+# Define needed variables
+CURRENT=$(npm pkg get version | sed 's/"//g')
+NEW_VERSION=$1
+BUMP_BRANCH="${NEW_VERSION}-version-bump"
+ISCSC_REMOTE=$(git remote -v | grep 'git@github.com:iScsc/iscsc.fr.git' | awk '{print $1}' | head --lines 1)
+
+# ----------- Version checking -----------
+
+# Check if targeted version is > than current
+echo "[+] Current version is '${CURRENT}'"
+if [ ! $(semver "${NEW_VERSION}" -r ">$CURRENT") ]; then
+	echo "[-] '${NEW_VERSION}'<='$CURRENT', '${NEW_VERSION}' isn't accepted as new version."
+	exit 1
+fi
+echo "[+] '${NEW_VERSION}'>'$CURRENT', '${NEW_VERSION}' is accepted as new version."
+
+# ----------- Git setup -----------
+
+# ...and checkout on main to create a version bump branch
+echo "[+] Checkout on ${ISCSC_REMOTE}/main"
+[ -z "$DRY_RUN" ] && git checkout ${ISCSC_REMOTE}/main
+echo "[+] switching to ${BUMP_BRANCH}"
+[ -z "$DRY_RUN" ] && git switch -c ${BUMP_BRANCH}
+
+# ----------- Version Bump -----------
+
+echo '[+] Bumping `frontend`'
+cd frontend
```

Comment:
to avoid `cd`ing in and out of directories, which is not the safest to land where you were before, you might want to use subshells :yum: 

instead of
```bash
cd somewhere
command
cd -  # or `cd back-somewhere
```
you can write
```bash
(
    cd somewhere
    # here you are somewhere
    command
)
# here you are back automatically where you were before
```

Reply:
done [e62c64f](https://github.com/iScsc/iscsc.fr/pull/27/commits/e62c64f01ab338028a0da19c73180e853ac71d8d)

Reply:
super neat :relieved: :muscle: